tests/vppinputoutput.h

```diff
@@ -19,7 +19,7 @@
 #include "common/log.h"
 #include "common/utils.h"
 #include "common/VaapiUtils.h"
-#include "common/videopool.h"
+#include <pooledframeallocator.h>
```

Comment:
We'd better not make this as public interface, since we may change this later, so just copy the header to yami-utils and call it.